data/boarding missions.txt

```diff
@@ -61,6 +61,160 @@ mission "Assisting Merchant (Large)"
 
 
 
+mission "Asteroid Course A"
+	assisting
+	minor
+	to offer
+		random < 10
+	source
+		government "Merchant" "Syndicate"
+		category "Light Freighter" "Interceptor"
+		not attributes "automaton"
+	destination "Earth"
+	on offer
+		conversation
+			`As you are approaching the <origin>, your radar sensors detect that the ship you are assisting is in the middle of a large asteroid's course! While in a normal flight this wouldn't cause much trouble, a ship without active shields and that has already suffered heavy damages could be completely destroyed.`
+			branch unarmed
+				"armament deterrence" == 0
+			choice
+				`	(Shoot towards the asteroid to destroy it.)`
+					goto fire
+				`	(Move your ship between the asteroid and the <origin>, then use your shields to protect it.)`
+					goto move
+				`	(Try to quickly repair the <origin>, so that it can move away from the asteroid's course.)`
+					goto repair
+				`	(Interrupt the docking maneuvers and fly away.)`
+					goto escape
+			label unarmed
+			choice
+				`	(Move your ship between the asteroid and the <origin>, then use your shields to protect it.)`
+					goto move
+				`	(Try to quickly repair the <origin>, so that it can move away from the asteroid's course.)`
+					goto repair
+				`	(Interrupt the docking maneuvers and fly away.)`
+			label escape
+			`	You turn your ship around and activate your thrusters, trying to get as far as you can from the disabled ship. A few seconds later, the asteroid tears the <origin> apart, killing all of its crew members.`
+				launch
```

Comment:
This should be `flee` or `depart`. Otherwise we're accepting this mission to go to Earth. Same story for the two below this. Since you're using `on decline`, you'd want to use `depart`, as that defers and doesn't trigger the `on decline`.